pallets/cash/src/core.rs

```diff
@@ -56,6 +56,17 @@ macro_rules! require {
     };
 }
 
+#[macro_export]
+macro_rules! must {
```

Comment:
same as require but "must" is easier to think?

Reply:
Must returns a `Result<(), Reason>` where `require!` is a unit value itself `if { ... };`. In some other code, I really want a `Result<(), Reason>` but I don't want to change the `require!` macro that we use in a lot of places.